test/kubernetes_unittest.cc

```diff
@@ -717,41 +717,72 @@ bool WaitForNewerCollectionTimestamp(const MetadataStore& store,
   return false;
 }
 
-TEST_F(KubernetesTest, KubernetesUpdater) {
+void TestKubernetesUpdater(bool cluster_level_metadata, bool service_metadata) {
```

Comment:
Please remove all tests for service_metadata, as we will eventually remove this option, and the default value is true.

Reply:
Done.